README.md

```diff
@@ -6,22 +6,67 @@ This project consists of a backend server and a React Native app that demonstrat
 
 * Dialpad for making outgoing calls to a phone number
 * Active call screen to interact with and control ongoing calls
+* Login flow for authenticating and authorizing users
 
 ### Planned Features
 
 * App to app calls
-* Login flow for authenticating and authorizing users
 * Registration and notifications for incoming calls
 
 ![dialer](https://user-images.githubusercontent.com/22135968/223575482-53f733b4-c828-4dd1-b100-7cc945a52c9f.jpg)
 ![active](https://user-images.githubusercontent.com/22135968/223575472-f112ab20-f11c-4f54-9432-3640ddbb0f1c.jpg)
+![login](https://user-images.githubusercontent.com/35968892/226741082-eb0ac83a-b16d-46a3-9ff3-a83167ed7267.png)
```

Comment:
Let's put this right before the dialpad screen. Also, can you please take a screenshot on iOS to be more consistent? Feel free to send the screenshot to me and I can clean up the edges as well.